include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2861,15 +2861,20 @@ namespace internal
         const Number *                         values_dofs,
         FEEvaluationData<dim, Number, true> &  fe_eval)
     {
-      const auto &       shape_info       = fe_eval.get_shape_info();
-      const auto &       shape_data       = shape_info.data.front();
-      const unsigned int face_no          = fe_eval.get_face_no();
-      const unsigned int face_orientation = fe_eval.get_face_orientation();
+      const auto &shape_info = fe_eval.get_shape_info();
+      const auto &shape_data = shape_info.data.front();
 
       if (shape_info.element_type == MatrixFreeFunctions::tensor_none)
         {
-          const std::size_t n_dofs     = shape_info.dofs_per_component_on_cell;
-          const std::size_t n_q_points = shape_info.n_q_points_faces[face_no];
+          Assert((fe_eval.get_dof_access_index() ==
+                    MatrixFreeFunctions::DoFInfo::dof_access_cell &&
+                  fe_eval.is_interior_face() == false) == false,
```

Comment:
Can we negate this statement?
```suggestion
          Assert(fe_eval.get_dof_access_index() !=
                    MatrixFreeFunctions::DoFInfo::dof_access_cell ||
                  fe_eval.is_interior_face() == true,
```